solutions/09/script.js

```diff
@@ -1,30 +1,63 @@
-function validateCardNumber(number) {
-    var regex = new RegExp("^[0-9]{16}$");
-    if (!regex.test(number))
-        return false;
+function validateCardNumber(value) {
+    // Accept only digits, dashes or spaces
+	if (/[^0-9-\s]+/.test(value)) return false;
 
-    return luhnCheck(number);
+	// The Luhn Algorithm. It's so pretty.
```

Comment:
Instead of commend, make separate function

Reply:
Done